tmva/sofie/src/SOFIE_common.cxx

```diff
@@ -35,6 +35,12 @@ ETensorType ConvertStringToType(std::string type){
    if(type == "float32" || type == "Float"){
      return ETensorType::FLOAT;
    }
+   else if(type == "int64" || type == "int"){
```

Comment:
`int` should be converted to INT32

Reply:
And above should not be `float` and not `Float` ?

Reply:
Hello sir, regarding the second comment if i change `Float` to `float` it gives an error for the gemm operator, so i have not made any changes in that, other all comments are addressed . Now you can review the PR once and if any changes are required i will do that. Thanks for the review!
![image](https://user-images.githubusercontent.com/84740927/182447847-d43ed68e-b324-467e-80b7-19a47b660d8a.png)

Though we still have the error in the infer function as mentioned below. Can you suggest the solution to fix it.
![image](https://user-images.githubusercontent.com/84740927/182448235-554c58e1-e00c-4012-a832-771db69b8af1.png)

Reply:
We should check in the ONNX parser (when running with verbose=true) if the name is `int64` or something else. I don't think we need to use the `int` case.